src/components/CellLineTable/SharedColumns.tsx

```diff
@@ -94,21 +96,29 @@ export const obtainLineColumn = {
     className: actionColumn,
     fixed: "right" as const,
     render: (orderLink: string) => {
-        return (
-            orderLink && (
-                <a
-                    key={orderLink}
-                    className={actionButton}
-                    href={orderLink}
-                    target="_blank"
-                    rel="noreferrer"
-                >
-                    <Flex>
-                        <TubeIcon />
-                        Obtain Collection
-                    </Flex>
-                </a>
-            )
+        const isDisabled = !orderLink;
+        const link = (
+            <a
+                key={orderLink}
+                className={classNames(actionButton, {
+                    [disabled]: isDisabled,
+                })}
+                href={isDisabled ? undefined : orderLink}
```

Comment:
does an empty string still make it clickable? just curious

Reply:
so could it just be `href={orderLink}` ? since isDisabled is just checking if it exists?